code-challenges.js

```diff
@@ -21,22 +21,78 @@ const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+// Pseudo code
+// Create a function named boilingPoint
+// Made a decision tree, use conditional to check boiling points
+// Input: any number
+// Output = 42 is below the boiling point, 350 is above the boiling point, and 212 is at boiling point
+// return the string if the number is below the boiling point, at the boiling point, or above boiling point.
+
+// const boilingPoint = (number) => {
```

Comment:
Please do not comment out the code block that you want me to evaluate. Great logic though!